src/modeling/layers/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .masked_patch_prediction_layer import MaskedPP
```

Comment:
Why the "." before "masked_patch_prediction_layer"?

Reply:
It's relative path. We can also do absolute path `src.modeling.layers.masked_patch_prediction_layer`.